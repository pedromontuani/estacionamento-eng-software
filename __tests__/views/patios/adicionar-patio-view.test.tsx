import React from 'react';
import {rendererCreate} from '../../utils';

import AddNewPatio from '../../../src/views/patios/adicionar-patio-view';

describe('Testing add new patio view', () => {
  test('should match snapshot', async () => {
    const tree = rendererCreate(<AddNewPatio />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
