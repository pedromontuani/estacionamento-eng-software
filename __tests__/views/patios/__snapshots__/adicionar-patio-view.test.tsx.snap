// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing add new patio view should match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "height": 1334,
      "padding": 24,
      "width": 750,
    }
  }
>
  <Text>
    Adicionar novo pátio
  </Text>
  <View
    style={
      Object {
        "paddingVertical": 24,
      }
    }
  >
    <View
      style={
        Object {
          "paddingVertical": 6,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "white",
            "borderRadius": 22,
            "elevation": 2,
            "height": 44,
            "justifyContent": "center",
            "paddingHorizontal": 8,
            "width": "100%",
          }
        }
      >
        <TextInput
          onChangeText={[Function]}
          placeholder="Código"
          value=""
        />
      </View>
    </View>
    <View
      style={
        Object {
          "paddingVertical": 6,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "white",
            "borderRadius": 22,
            "elevation": 2,
            "height": 44,
            "justifyContent": "center",
            "paddingHorizontal": 8,
            "width": "100%",
          }
        }
      >
        <TextInput
          keyboardType="numeric"
          onChangeText={[Function]}
          placeholder="Número de vagas"
          value=""
        />
      </View>
    </View>
    <View
      style={
        Object {
          "paddingVertical": 6,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "white",
            "borderRadius": 22,
            "elevation": 2,
            "height": 44,
            "justifyContent": "center",
            "paddingHorizontal": 8,
            "width": "100%",
          }
        }
      >
        <TextInput
          keyboardType="numeric"
          onChangeText={[Function]}
          placeholder="Valor hora"
          value=""
        />
      </View>
    </View>
  </View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#1976d2",
        "borderRadius": 20,
        "elevation": 5,
        "height": 30,
        "justifyContent": "center",
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 14,
          "fontWeight": "500",
        }
      }
    >
      Salvar
    </Text>
  </View>
</View>
`;
